shared-kcp/sharedkcp.kubeconfig

```diff
@@ -0,0 +1,55 @@
+apiVersion: v1
+clusters:
+- cluster:
+    insecure-skip-tls-verify: true
+    server: https://api.ci-ln-01qrpgt-72292.origin-ci-int-gce.dev.rhcloud.com:6443
+  name: api-ci-ln-01qrpgt-72292-origin-ci-int-gce-dev-rhcloud-com:6443
+- cluster:
+    server: https://kcp-stable.apps.kcp-internal-1.52p3.p1.openshiftapps.com/clusters/root:rh-sso-15826143
+  name: kcp-stable
+- cluster:
+    server: https://kcp-unstable.apps.kcp-internal-1.52p3.p1.openshiftapps.com/clusters/root:rh-sso-15826143
+  name: kcp-unstable
+- cluster:
+    server: https://kcp-stable.apps.kcp-internal-1.52p3.p1.openshiftapps.com/clusters/rh-sso-15826143:plsvc
+  name: workspace.kcp.dev/personal/plsvc
+contexts:
+- context:
+    cluster: api-ci-ln-01qrpgt-72292-origin-ci-int-gce-dev-rhcloud-com:6443
+    namespace: default
+    user: kube:admin/api-ci-ln-01qrpgt-72292-origin-ci-int-gce-dev-rhcloud-com:6443
+  name: default/api-ci-ln-01qrpgt-72292-origin-ci-int-gce-dev-rhcloud-com:6443/kube:admin
+- context:
+    cluster: kcp-stable
+    namespace: default
+    user: oidc
+  name: kcp-stable
+- context:
+    cluster: kcp-unstable
+    namespace: default
+    user: oidc
+  name: kcp-unstable
+- context:
+    cluster: workspace.kcp.dev/personal/plsvc
+    user: oidc
+  name: workspace.kcp.dev/personal/plsvc
+current-context: workspace.kcp.dev/personal/plsvc
+kind: Config
+preferences: {}
+users:
+- name: kube:admin/api-ci-ln-01qrpgt-72292-origin-ci-int-gce-dev-rhcloud-com:6443
+  user:
+    token: sha256~5k1bZMZzeFomHBsugloXbBSPq-5VxOPABUmfmSoA8cc
+- name: oidc
+  user:
+    exec:
+      apiVersion: client.authentication.k8s.io/v1beta1
+      args:
+      - oidc-login
```

Comment:
We use oidc-login, but there's no mention of it in the `README.md`, nor is the script making sure that it's installed. If I was asked to execute this script as a newcomer, I would have no idea where to start.

Reply:
I think this is a pre-requisite for the kcp onboarding doc. I would assume that someone trying to use this script would have already installed this and other resources required for kcp shared instance to work (oidc login, kubelogin, kcp plugin etc)